llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -182,3 +200,61 @@ def _llm_response_to_plan(response: LLMResponse, task: Task) -> Plan:
             action = f"({op} {objects_str})"
             plan.append(action)
         return plan
+
```

Comment:
can we make all of these new methods private, since we don't expect to use them outside of the approach itself? (other than in tests, where we can do the disable protected-access trick)